backend/src/nodes/nodes/image_filter/blur_lens.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import annotations
+
+import math
+import numpy as np
+from scipy import signal
+from functools import reduce
+
+from . import category as ImageFilterCategory
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput, SliderInput
+from ...properties.outputs import ImageOutput
+
+# lens blur adapted from: https://github.com/NatLee/Blur-Generator
```

Comment:
The license of that project requires us to credit the author, so let's add a little note.

```suggestion
# Lens blur adapted from: https://github.com/NatLee/Blur-Generator
# MIT License; Copyright (c) 2022 Nat Lee
```